brambling/utils/model_tables.py

```diff
@@ -482,8 +485,13 @@ def order_placed_by(self, obj):
             return "{} ({})".format(person.get_full_name(), person.email)
         return obj.order.email
 
-    def order_balance(self, obj):
-        return format_money(obj.order_balance or 0, self.event.currency)
+    def paid(self, obj):
+        paid = 0
+        for item in obj.items:
+            paid += item.price
+            for discount in item.discounts.all():
+                paid -= discount.savings()
+        return format_money(paid, self.event.currency)
```

Comment:
I feel like this could be a little more compact, e.g.

```python
paid = sum(i.price for i in obj.items)
paid -= sum(d.savings() for d in obj.items.values_list('discounts', flat=True) if d)
```
Which is possibly a bad/nonstandard way to query up those discounts but it's what I came up quickly.. maybe you know of a better way?

I mean, we are adding up numbers. Surely there's more efficient way than nested for loops?